recipes/mobilesam/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mobilesam" %}
+{% set version = "1.0.0.20231030.12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ChaoningZhang/MobileSAM.git
+  git_rev: 12d80d4e32b277de299130d8ce28cc949fb54b6c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - pytorch
+    - timm
```

Comment:
```suggestion
```

Reply:
it s eems to work without it.

during setup, only the `setup.py` file is called, so you can inspect it to look at dependencies